sky/global_user_state.py

```diff
@@ -101,6 +101,9 @@ def create_table(cursor, conn):
 
     db_utils.add_column_to_table(cursor, conn, 'clusters', 'cluster_hash',
                                  'TEXT DEFAULT null')
+
+    db_utils.add_column_to_table(cursor, conn, 'cluster_history', 'status',
+                                 'TEXT DEFAULT null')
```

Comment:
It seems to me this column is not necessary for the `cluster_history` table as we can always do a `join` for the `cluster_history` and the `clusters` table to get the status. Do you think we should do the something like the following whenever we need the status instead?
```
SELECT * 
FROM cluster_history
LEFT OUTER JOIN clusters ON cluster_history.cluster_hash=clusters.cluster_hash
```

Reply:
Should we only show status for clusters that are not terminated, so clusters that are in both the `clusters` and `cluster_history` table?

Reply:
Actually never mind, lets show terminated as well, which will be a result of when the sql join query returns nothing.